.gitattributes

```diff
@@ -0,0 +1,2 @@
+/tests/debugger/bug00627-symlink.inc -text -filter
+/tests/develop/bug00744.phpt -text -filter
```

Comment:
Why is this necessary?

Reply:
these files test special formatting (ie. no end EOL or `\r\n` EOL), so this prevent any text filters to be applied